src/main/java/org/zeromq/ZStar.java

```diff
@@ -520,7 +520,6 @@ public void run(final Object[] train, final ZContext chef, final Socket mic)
                         }
                         catch (Exception e) {
                             // really ?
-                            e.printStackTrace();
```

Comment:
I'm curious about this one. Swallowing exceptions is usually bad news, especially unspecific exceptions. Probably better to remove the try/catch altogether and let exceptions happen, if this is a "this should never happen" scenario. Or we could wrap it in some kind of ZMQ exception wrapper class.